app/src/main/java/tech/miam/coursesUDemoApp/features/miam/template/MiamManager.kt

```diff
@@ -19,13 +21,15 @@ import kotlinx.coroutines.withContext
 import org.koin.core.component.KoinComponent
 import org.koin.core.component.inject
 import tech.miam.coursesUDemoApp.basket.BasketEvent
+import tech.miam.coursesUDemoApp.data.models.Attributes
 import tech.miam.coursesUDemoApp.data.models.Product
 import tech.miam.coursesUDemoApp.features.miam.template.MiamSdkHelper.pushProductToRetailer
 import tech.miam.coursesUDemoApp.features.products.ProductsRepository
 import tech.miam.coursesuui.config.MiamTemplateManager
 import timber.log.Timber
 
-object MiamSdkHelper : CoroutineScope by CoroutineScope(Dispatchers.Main), KoinComponent, BasketPublisher, BasketSubscriber {
+object MiamSdkHelper : CoroutineScope by CoroutineScope(Dispatchers.Main), KoinComponent, BasketPublisher,
```

Comment:
still using koin ?

Reply:
removed it from this file (still used in other files)